images/linux/scripts/installers/java-tools.sh

```diff
@@ -15,6 +15,10 @@ function javaTool {
     fi
 }
 
+toolset="$INSTALLER_SCRIPT_FOLDER/toolset.json"
+JAVA_VERSIONS_LIST=$(jq -r '.java.versions | .[]' $toolset)
+JAVA_DEFAULT=$(jq -r '.java.default' $toolset)
```

Comment:
nitpick: we don't use `JAVA_DEFAULT` var anywhere, I think we should rename to `DEFAULT_JDK_VERSION`.